dev-scripts/cleanup.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
worth making this a python script so we can run it on Windows too?

shutil.rmtree() # to delete folders
os.remove() # to delete files
glob.glob() # to find files

Reply:
well, the windows generates different folder, but yeah, that sounds good.